### ISSUE TITLE
Wrong path to containers.conf

### ISSUE THREAD

**Issue by vrandr** [containers/libpod#6163]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description:**

-  If you use a `$HOME/.config/containers/libpod.conf` configuration file you'll receive a wrong Debug Message on were to put the new `containers.conf`

**Steps to reproduce the issue:**

1. Setup a configuration file `$HOME/.config/containers/libpod.conf`

2. Run `podman ps --log-level=debug`

**Describe the results you received:**

1. `DEBU[0000] Found deprecated file $HOME/.config/containers/libpod.conf, please remove. Use $HOME/.config/.config/containers/containers.conf to override defaults.`
2. `DEBU[0000] Ignoring lipod.conf EventsLogger setting "none". Use containers.conf if you want to change this setting and remove libpod.conf files.`

**Describe the results you expected:**

1. The path to `containers.conf` should not include two `.config/.config` directories
2. Typo `lipod.conf ` instead of `libpod.conf`

**Output of `podman version`:**

```
Version:            1.9.1
RemoteAPI Version:  1
Go Version:         go1.10.1
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

```
debug:
  compiler: gc
  gitCommit: ""
  goVersion: go1.10.1
  podmanVersion: 1.9.1
host:
  arch: amd64
  buildahVersion: 1.14.8
  cgroupVersion: v1
  conmon:
    package: 'conmon: /usr/libexec/podman/conmon'
    path: /usr/libexec/podman/conmon
    version: 'conmon version 2.0.15, commit: '
  cpus: 2
  distribution:
    distribution: ubuntu
    version: "18.04"
  eventLogger: file
  hostname: hostname
  idMappings:
    gidmap:
    - container_id: 0
      host_id: uid
      size: 1
    - container_id: 1
      host_id: 820896
      size: 65536
    uidmap:
    - container_id: 0
      host_id: uid
      size: 1
    - container_id: 1
      host_id: 820896
      size: 65536
  kernel: 4.15.0-88-generic
  memFree: 206110720
  memTotal: 4135727104
  ociRuntime:
    name: runc
    package: 'cri-o-runc: /usr/lib/cri-o-runc/sbin/runc'
    path: /usr/lib/cri-o-runc/sbin/runc
    version: 'runc version spec: 1.0.1-dev'
  os: linux
  rootless: true
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: 'slirp4netns: /usr/bin/slirp4netns'
    version: |-
      slirp4netns version 0.4.3
      commit: unknown
  swapFree: 3107450880
  swapTotal: 4133482496
  uptime: 1507h 51m 10.19s (Approximately 62.79 days)
registries:
  search:
  - docker.io
  - quay.io
store:
  configFile: /home/user/.config/containers/storage.conf
  containerStore:
    number: 5
    paused: 0
    running: 5
    stopped: 0
  graphDriverName: vfs
  graphOptions: {}
  graphRoot: /home/user/.local/share/containers/storage
  graphStatus: {}
  imageStore:
    number: 16
  runRoot: /run/user/uid/containers
  volumePath: /home/user/.local/share/containers/storage/volumes
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman/unknown,now 1.9.1~1 amd64 [installed]
```

**Comment by vrothberg** [comment]
Thanks for the report! https://github.com/containers/libpod/pull/6175 will fix the issue for Podman 1.9.

**Comment by vrothberg** [comment]
Closing as this is fixed in master and backported to 1.9. @mheon will release a new patch release for 1.9 soon.
